draft-ietf-acme-acme.md

```diff
@@ -1972,6 +1997,11 @@ domain by verifying that the resource was provisioned as expected.
 5. Verify that key authorization provided by the server matches the token for
    this challenge and the client's account key.
 
+The server MAY follow redirects when dereferencing the URI.
```

Comment:
So your intent is that the server MUST follow redirects? I'm fine with that, just trying to clarify the spec since it doesn't say either way.